### PR TITLE
Revised the null-coalescing operator article

### PR DESCRIPTION
**Pull request by pkulikov**
Contributes to #10741 
Supported by dotnet/samples#968

### CODE REVIEW

File: docs/csharp/language-reference/operators/conditional-operator.md [hunk 1]
```diff
@@ -80,14 +80,14 @@ The conditional operator cannot be overloaded.
 
 ## C# language specification
 
-For more information, see the [Conditional operator](~/_csharplang/spec/expressions.md#conditional-operator) section of the [C# language specification](../language-specification/index.md).
+For more information, see the [Conditional operator](~/_csharplang/spec/expressions.md#conditional-operator) section of the [C# language specification](~/_csharplang/spec/introduction.md).
 
 ## See also
 
 - [C# Reference](../index.md)
 - [C# Programming Guide](../../programming-guide/index.md)
```
Comment (pkulikov) [inline docs/csharp/language-reference/operators/conditional-operator.md]: @BillWagner is that link to the programming guide still useful? Are those used (and if used, the final result is satisfying?)? Given future restructures of the C# doc set, would it be fine to remove that link from the **See also** sections already now?

**Comment by BillWagner** [comment]
closing and reopening for a fresh build

File: docs/csharp/language-reference/operators/conditional-operator.md (hunk 1 above)
Comment (BillWagner) [reply docs/csharp/language-reference/operators/conditional-operator.md]: I agree that we can remove these.

The data shows they aren't clicked on much.

Comment (pkulikov) [reply docs/csharp/language-reference/operators/conditional-operator.md]: @BillWagner thanks for the check. I'll submit a separate PR with those sections clean-up, then.

**Review by BillWagner — APPROVED** [review]
Thanks again for all this work @pkulikov 

This is another great set of improvements. I'll :shipit: now.

Thanks again!

**Pull request merged**